packages/grid/_modules_/grid/components/icons/index.tsx

```diff
@@ -11,4 +11,9 @@ export const ArrowDownwardIcon = createSvgIcon(
   'ArrowDownward',
 );
 
+export const FilterIcon = createSvgIcon(
```

Comment:
I think that we should retain the original name. At the minimum to remove any discussion around best naming, and at best, to keep the reference to the original icon, in case somebody looking in the source want to find it from @material-ui/icons or if the material design icon updates in the future.

```suggestion
export const FilterListIcon = createSvgIcon(
```